### PR TITLE
Nav design tweaks

### LINKED ISSUE #429: The ACMCSUF Logo disappears when navigating to a URL with multi-level subdirectory
### What happened?

For example, when I go to [https://acmcsuf.com/paths/ToTheMoon](https://acmcsuf.com/paths/ToTheMoon), the logo on the navigation bar is gone.
<br />

### PR DESCRIPTION
**Pull request by ghost**
- Reverted to right-leaning nav for desktop
- h3 properly clamps for 280px vw
- Horizontal padding on mobile changed to 16px
- Added 'chapter' to badge
- added 'at' to logo wording
- changed general badge alt tag

### CODE REVIEW

File: src/lib/components/sections/navbar.svelte [hunk 1]
```diff
@@ -31,44 +31,46 @@
   <div class="container">
     <div class="logo-container">
       <a href="/" class="logo">
-        <img src="assets/badges/general.svg" class="badge" alt="acmCSUF logo" />
-        <h3>CSUF</h3>
+        <img src="assets/badges/general.svg" class="badge" alt="acmCSUF badge" />
+        <h3 class="brand-header">at <b>CSUF</b></h3>
```
Comment (ghost) [inline src/lib/components/sections/navbar.svelte:35]: Bold tag works the same functionally as CSS .brand-em. Might depreciate that CSS style.

File: src/lib/components/sections/navbar.svelte [hunk 2]
```diff
@@ -31,44 +31,46 @@
...
-    <ul class="pages">
-      {#each navItems as { title, path } (path)}
-        <li>
-          <a
-            on:click={() => handleClose()}
-            href={path}
-            class="brand-header size-sm"
-            aria-current={path === $page.url.pathname}
-            sveltekit:prefetch
+    <section>
```
Comment (ghost) [inline src/lib/components/sections/navbar.svelte:39]: Section used to put nav items to the right like previously

File: src/lib/components/sections/navbar.svelte [hunk 3]
```diff
@@ -93,17 +95,15 @@
       display: flex;
       align-items: center;
       justify-content: space-between;
-      padding: 0 24px;
+      padding: 0 16px;
```
Comment (ghost) [inline src/lib/components/sections/navbar.svelte:98]: Lowered padding to help w/ scaling to 280px vw

File: src/lib/components/sections/navbar.svelte [hunk 4]
```diff
@@ -165,22 +171,14 @@
...
-          }
+        .logo-container .logo .badge {
+          height: 48px;
         }
 
         .pages {
           position: fixed;
           flex-direction: column;
-          justify-content: space-evenly;
+          justify-content: center;
```
Comment (ghost) [inline src/lib/components/sections/navbar.svelte:182]: Justify center to condense the look of the buttons. Enough spacing like this already due to `gap: 64px` in line 127

File: src/lib/components/sections/navbar.svelte [hunk 5]
```diff
@@ -112,44 +112,50 @@
           }
 
           h3 {
-            font-size: clamp(1.25rem, 1.1635rem + 0.3846vw, 1.625rem);
+            font-size: clamp(1.25rem, 1.2039rem + 0.2632vw, 1.625rem);
```
Comment (ghost) [inline src/lib/components/sections/navbar.svelte:116]: None of the CSS sizing tags are used here because the Logo text scales to very specific px sizes

**Review by EthanThatOneKid — APPROVED** [review]
LGTM!

**Review by jaasonw — CHANGES_REQUESTED** [review]
can you run `npm run all` and then commit the result to fix the formatting check

**Comment by ghost** [comment]
> can you run `npm run all` and then commit the result to fix the formatting check

i did lol

EDIT: oh wait did it on the wrong branch lmao

**Comment by jaasonw** [comment]
> > can you run `npm run all` and then commit the result to fix the formatting check
> 
> i did lol

image

run it and commit this file

**Comment by ghost** [comment]
> > > can you run `npm run all` and then commit the result to fix the formatting check
> > 
> > 
> > i did lol
> 
> image
> 
> run it and commit this file

Read my edit lmfao

**Comment by jaasonw** [comment]
> > > > can you run `npm run all` and then commit the result to fix the formatting check
> > > 
> > > 
> > > i did lol
> > 
> > 
> > image
> > run it and commit this file
> 
> Read my edit lmfao

nice1

**Review by jaasonw — APPROVED** [review]

**Pull request merged**